synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -982,8 +982,10 @@ StateBone_Context::event_mouse_release_handler(const Smach::event& x)
 			}
 			else{
 				if(duck){
-					_on_signal_change_active_bone(duck->get_value_desc().get_parent_value_node());
-					get_work_area()->set_active_bone_value_node(duck->get_value_desc().get_parent_value_node());
+					if(duck->get_value_desc().is_parent_desc_declared() && ValueNode_Bone::Handle::cast_dynamic(duck->get_value_desc().get_parent_desc().get_value_node())){
```

Comment:
what about store `duck->get_value_desc().get_parent_desc().get_value_node()` in a variable? :)

Reply:
Yea okay! 😃

Reply:
In this  scope I mean. You use it 3 times and it's so long to read lol

Reply:
Haha yea xD

Reply:
I will wait for changes then. ^__^